public/js/controllers/traffic_graphs_controller.js

```diff
@@ -283,7 +312,14 @@ export class TrafficGraphsController {
             .shapeWidth(12)
             .orient("vertical")
             .scale(colorReceived)
-            .labels(operators);
+            .labels(operators)
+            .on("cellover", function () {
```

Comment:
```suggestion
            .on("cellhover", function () {
```

Reply:
For quick monitoring, it would be great to draw the selected network on cellHover even without clicking and reset to all otherwise.